content/en/tracing/profiling/java.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Profiling Java Applications
+kind: documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+The Datadog Profiling library supports the Java JRE 11 and higher of both Oracle JDK and OpenJDK. Early-access versions of Java aren't supported.
+Java profiling and tracing are shipped within the same `dd-java-agent.jar` library. In order to use tracing or profiling, you must enable these functions. To enable tracing, follow the [enable tracing documentation][1].
+
+**Note**: Some specific builds will work with Datadog profiling if they support [Java Flight Recorder][2].
+
+## Getting Started
+
+To begin profiling applications written in any JVM based runtime:
+
+1. Download `dd-java-agent.jar`, which contains the Agent class files:
```

Comment:
Needs update for the imminent release of the open source lib.

Reply:
Since we talk about both the java agent and the system agent, we should be explicit in each case about which we mean.

Reply:
I think we don't reference the "system agent".. but i might be wrong.. we usuall refer to it as "the Agent" as there is only one to rule themmmm allll. Would you have a link where we talk about the system agent ?

Reply:
Agreed, normally we just say "the agent". In most contexts this is right and there's no need to add more words :-) It's confusing here though, because "the agent" could refer to the "normal" agent, or the java agent which is the tracer and profiler.

Reply:
The "java Agent" is a wording that should be avoided, i'm going to fix references to it to avoid confusion